dotCMS/src/main/java/com/dotcms/rest/api/v1/workflow/WorkflowResource.java

```diff
@@ -625,9 +633,11 @@ public final Response deleteAction(@Context final HttpServletRequest request,
     } // deleteAction
     
     /**
+     * Todo: change the signature to be align with the rest implementation such as: reorderAction
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Complete the task associated to this TODO comment. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1135)